src/dsimhost_fpga.py

```diff
@@ -66,13 +66,23 @@ def __init__(self, name, scale_register, control_register):
         self.name = name
         self.scale_register = scale_register
         self.control_register = control_register
+        self.tgv_select_field = 'tvg_select' + self.name
 
+    @property
+    def output_type(self):
+        """Curently selected output type"""
+        #if
```

Comment:
You need to bring back the if statement, but instead of printing to return 'test_vectors' or return 'signal', i.e. use the same values you would when setting in select_output()